packages/react-server/src/logger/logger.ts

```diff
@@ -1,50 +1,87 @@
-import {Context} from 'koa';
+/* eslint-disable no-process-env */
+import {Context, Request} from 'koa';
 import chalk from 'chalk';
 import {KoaNextFunction} from '../types';
 
 export const LOGGER = Symbol('logger');
-const PREFIX = chalk`{underline sidecar} ⁓ `;
+const PREFIX = chalk`{underline [React Server]}  `;
 
-interface LoggerOptions {
-  level?: Verbosity;
```

Comment:
The `level` option dictates the `verbosity` of the logger. The `verbosity` is kind of a lie, it's more like `noopLogger` or `logger`. Apps should get logging for free both in production and dev environments. When it is actually verbose, we can re-evaluate the need for this.